app/lib/features/home/widgets/sidebar_widget.dart

```diff
@@ -12,31 +12,36 @@ import 'package:go_router/go_router.dart';
 
 class SidebarWidget extends ConsumerWidget {
   final NavigationRailLabelType labelType;
+  final StatefulNavigationShell navigationShell;
 
   const SidebarWidget({
     super.key = Keys.mainNav,
     required this.labelType,
+    required this.navigationShell,
   });
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final sidebarNavItems = ref.watch(sidebarItemsProvider(context));
-    final selectedSidebarIndex =
-        ref.watch(currentSelectedSidebarIndexProvider(context));
+    // final selectedSidebarIndex =
```

Comment:
Remove this line instead of commenting :)

Reply:
Ya sure